recipes/ucc/meta.yaml

```
@@ -0,0 +1,53 @@
+package:
+  name: ucc
+  version: v1.2.0
+
+source:
+  url: https://github.com/openucx/ucc/archive/refs/tags/v1.2.0.tar.gz
+  sha256: c1552797600835c0cf401b82dc89c4d27d5717f4fb805d41daca8e19f65e509d
+  patches:
+    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch  # [(cuda_compiler_version or "None").startswith("12")]
```

Is this upstream PR ( https://github.com/openucx/ucc/pull/847 )? If so, maybe we can add a comment?

Also would suggest applying this patch in all cases as this is what will happen when we upgrade to the new release

```suggestion
  patches:
    # Apply upstream fix for CUDA versions pre-12.2
    # xref: https://github.com/openucx/ucc/pull/847
    - 0001-EC-CUDA-fix-build-with-12.2-and-older-arch-847.patch
```